### PR TITLE
Addition of SingleEleNonIso HLT path of pT threshold 115 GeV to Phase2 HLT menu

### PR DESCRIPTION
**Pull request by swagata87**
#### PR description:
This PR adds a new HLT path (L1-seeded single electron non-isolated path) to the Phase2 regular menu and Phase2 timing menu. Performance of this new path (efficiency, rate etc) was presented in [this talk](https://indico.cern.ch/event/1324703/contributions/5574449/attachments/2712507/4710934/EleNonIso_ph2HLT.pdf) at the [HLT upgrade meeting of 12 Sept 2023](https://indico.cern.ch/event/1324703/). This is in-line with the plans of EGamma POG as outlined in [this talk](https://indico.cern.ch/event/1222160/contributions/5141648/attachments/2566172/4424146/HLT_UW_2022.pdf) (slide 17) at the [6th HLT upgrade workshop in December 2022](https://indico.cern.ch/event/1222160/).

#### PR validation:
1) `runTheMatrix.py -l 24834.911` ran successfully.
2) `cmsDriver.py Phase2 -s HLT:75e33 --processName=HLTX --conditions auto:phase2_realistic_T21 --geometry Extended2026D95 --era Phase2C17I13M9  --customise SLHCUpgradeSimulations/Configuration/aging.customise_aging_1000 --filein=/store/relval/CMSSW_13_3_0_pre2/RelValZpToEE_m6000_14TeV/GEN-SIM-DIGI-RAW/PU_131X_mcRun4_realistic_v6_2026D98PU200-v1/2580000/015b1c6a-25be-4a3c-bca3-ae047eb03fbb.root --inputCommands='keep *, drop *_hlt*_*_HLT, drop triggerTriggerFilterObjectWithRefs_l1t*_*_HLT' -n 10 --nThreads 1 --no_exec` and then `cmsRun Phase2_HLT.py` ran fine, and this is part of the output:
```
HLT-Report   HLT #  WasRun     L1S     Pre     HLT   %L1sPre    Rate  RateHi  Errors Name
HLT-Report      36      10      10      10       6  60.00000     0.0     0.0       0 HLT_Ele32_WPTight_L1Seeded
HLT-Report      37      10      10      10       9  90.00000     0.0     0.0       0 HLT_Ele115_NonIso_L1Seeded
HLT-Report      38      10      10      10       6  60.00000     0.0     0.0       0 HLT_Ele26_WP70_L1Seeded
HLT-Report      39      10      10      10       9  90.00000     0.0     0.0       0 HLT_Photon108EB_TightID_TightIso_L1Seeded
HLT-Report      40      10      10      10       8  80.00000     0.0     0.0       0 HLT_Photon187_L1Seeded
HLT-Report      41      10      10      10       5  50.00000     0.0     0.0       0 HLT_DoubleEle25_CaloIdL_PMS2_L1Seeded
HLT-Report      42      10      10      10       3  30.00000     0.0     0.0       0 HLT_DoubleEle23_12_Iso_L1Seeded
HLT-Report      43      10      10      10       7  70.00000     0.0     0.0       0 HLT_Diphoton30_23_IsoCaloId_L1Seeded
HLT-Report      48      10      10      10      10 100.00000     0.0     0.0       0 MC_Ele5_Open_Unseeded

```

More detailed checks of efficiency of the new path were made in `CMSSW_13_1_X` by running on this dataset[*] as described in [the talk](https://indico.cern.ch/event/1324703/contributions/5574449/attachments/2712507/4710934/EleNonIso_ph2HLT.pdf). 

[*] `/ZprimeToEE_M-6000_TuneCP5_14TeV-pythia8/Phase2Spring23DIGIRECOMiniAODPU200_Trk1GeV_131X_mcRun4_realistic_v5-v1/GEN-SIM-DIGI-RAW-MINIAOD`

Backport of this PR is not necessary.

Tagging HLT upgrade convenors (@rovere, @SohamBhattacharya), EGM-HLT convenors (@DebabrataBhowmik, @ravindkv), EGM convenors (@cms-sw/egamma-pog-l2).

### CODE REVIEW

**Comment by mmusich** [comment]
@cmsbuild, please test

File: HLTrigger/Configuration/python/HLT_75e33_timing_cff.py [hunk 1]
```diff
@@ -136,10 +131,6 @@
 fragment.load("HLTrigger/Configuration/HLT_75e33/paths/L1T_TkIsoEm22TkIsoEm12_cfi")
 fragment.load("HLTrigger/Configuration/HLT_75e33/paths/L1T_TkIsoEm36_cfi")
 fragment.load("HLTrigger/Configuration/HLT_75e33/paths/L1T_TripleTkMuon_5_3_3_cfi")
-fragment.load("HLTrigger/Configuration/HLT_75e33/paths/MC_BTV_cfi")
-fragment.load("HLTrigger/Configuration/HLT_75e33/paths/MC_Ele5_Open_L1Seeded_cfi")
-fragment.load("HLTrigger/Configuration/HLT_75e33/paths/MC_Ele5_Open_Unseeded_cfi")
-fragment.load("HLTrigger/Configuration/HLT_75e33/paths/MC_JME_cfi")
```
Comment (mmusich) [inline HLTrigger/Configuration/python/HLT_75e33_timing_cff.py:142]: are the removals of 

```
fragment.load("HLTrigger/Configuration/HLT_75e33/paths/MC_BTV_cfi")
fragment.load("HLTrigger/Configuration/HLT_75e33/paths/MC_JME_cfi")
```

intentional here?

Comment (rovere) [reply HLTrigger/Configuration/python/HLT_75e33_timing_cff.py:142]: Ciao @mmusich the `MC_*` paths are not part of the `timing` version of the simplified menu. In #42759 those paths were removed from the `schedule`, so removing their `import` is a nice cleanup, IMHO.

**Comment by mmusich** [comment]
type egamma

File: HLTrigger/Configuration/python/HLT_75e33_timing_cff.py (hunk 1 above)
Comment (mmusich) [reply HLTrigger/Configuration/python/HLT_75e33_timing_cff.py:142]: thanks for the clarification... still learning here :D

**Comment by mmusich** [comment]
@rovere and @SohamBhattacharya let us know if you have any further comments on this PR.

**Comment by SohamBhattacharya** [comment]
> @rovere and @SohamBhattacharya let us know if you have any further comments on this PR.

Swagata informed us that she's checking the paths in 13_3 RelVals. I'd wait for her conclusion.

**Comment by SohamBhattacharya** [comment]
Hi @mmusich , no further comments on this PR.
@swagata87 has found the efficiency of Ele115 to be a bit lower than expected in 13_3, but that seems to be because of a different issue which affects all Phase-2 electron paths, and not to do with this PR itself. That's being investigated and a fix will come later.
So this PR is fine from the pov of hlt-upgrade.

**Comment by mmusich** [comment]
> no further comments on this PR.

OK, thanks.

> has found the efficiency of Ele115 to be a bit lower than expected in 13_3, but that seems to be because of a different issue which affects all Phase-2 electron paths, and not to do with this PR itself. 

would be good to open a gitHub issue with the details of the problem (@swagata87).

**Comment by swagata87** [comment]
GH issue is opened here: https://github.com/cms-sw/cmssw/issues/42850

**Comment by mmusich** [comment]
+hlt
   * see https://github.com/cms-sw/cmssw/pull/42819#issuecomment-1731744601
   * additional investigation on electron paths deficit is descoped from this PR and followed up at https://github.com/cms-sw/cmssw/issues/42850

**Comment by rappoccio** [comment]
+1

**Pull request merged**